### ISSUE TITLE
Wrong warning "Useless let-binding"

### ISSUE THREAD

**Issue by david-maison** [OCamlPro/typerex-lint#154]
Tested on tag ``alpha.0.1`` (commit aa0e597)

With the following file ``for_ocp.ml``:
```
let f () = 0, ""

let _ =
  let a, b = f () in
  b, a
```

We have this output:
```
% ocp-lint --pwarning --no-db-cache
Scanning files in project "."...
Found '1' file(s)
Running analyses... 1 / 1
Mergin database...
File "for_ocp.ml", line 1:
  Warning 1: plugin_file_system/interface_missing/missing_interface SEVERITY 4
  Missing interface for 'for_ocp.ml'.
File "for_ocp.ml", line 4, characters 2-27:
  Warning 1: plugin_patch/sempatch_lint/LetIdentity SEVERITY 2
  Useless let-binding: use 'f ()' without let-binding.
== New Warnings ==
  * 1 file(s) were linted
  * 2 warning(s) were emitted:
      * 1 warning(s) raised by "plugin_patch"/"sempatch_lint.1"/warning #1
      * 1 warning(s) raised by "plugin_file_system"/"interface_missing.1"/warning #1
  * 0 file(s) couldn't be linted
```

Since we swap the pair returned by the function ``f``, the let-binding seems to not be useless (except if we change the function ``f``, but I do not think we should recommend this).
